privacyidea/lib/crypto.py

```diff
@@ -193,7 +195,13 @@ def verify_with_pepper(passwordhash, password):
     # get the password pepper
     password = password or ""
     key = get_app_config_value("PI_PEPPER", "missing")
-    success = passlib.hash.pbkdf2_sha512.verify(key + password, passwordhash)
+
+    success = False
+    if passwordhash.startswith("$argon2"):
```

Comment:
We should add a not to the READ_BEFORE_UPDATE that passwords should be updated to use the new hash

Reply:
Will do.

Will add to Changelog, so we can remeber to note this in our email communication.

Reply:
Done in 4156eea9